ethereum_history_api/oracles/src/util/object.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-argument */
```

Comment:
Can we do this:
```
  pathArray.reduce((acc: T, key: string, i: number) => {
```
instead?

Reply:
Rewrote this function completely. It's now readable and pleases the linter completely